daemons/controld/controld_callbacks.c

```diff
@@ -354,27 +354,6 @@ peer_update_callback(enum crm_status_type type, crm_node_t * node, const void *d
     controld_trigger_fsa();
 }
 
-void
-crmd_cib_connection_destroy(gpointer user_data)
```

Comment:
Actually I was thinking about that, and I think callbacks.c is not a logical separation. I think code should be separated by its sphere of influence rather than the type of code it is. For example it would be silly to have a recursive_functions.c. I think longer term, I'd rather see the stuff in callbacks.c separated into thematic files. Also, callbacks tend to be used in one spot, so they can be static if grouped with that.

Reply:
That makes good sense to me